### PR TITLE
Add numero2/contao-opengraph3

### PR DESCRIPTION
**Pull request by bennyborn**

### CODE REVIEW

**Comment by fritzmg** [comment]
The whitelist contains more changes than necessary. Please create a separte PR, if you want to change or re-order existing whitelist entries.

File: meta/numero2/contao-opengraph3/logo.svg [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	<path class="st3" d="M48.9,45.5c1.9,0,3.8,0,5.7,0c-0.1,2-0.4,4.1-0.8,6c-2.7-0.5-5.4-0.7-8.1-0.8c0-0.7,0-1.4,0-2.1
+		C47.2,48,48.4,46.9,48.9,45.5z"/>
+	<path class="st3" d="M57.5,45.5c3,0,5.9,0,8.9,0c0,3.3,0,6.6,0,9.9c-3.1-1.4-6.4-2.5-9.7-3.3C57.1,49.9,57.4,47.7,57.5,45.5z"/>
+	<path class="st3" d="M35.5,54.2c2.5-0.4,5-0.7,7.4-0.8c0,4.7,0,9.4,0,14.1C39.5,63.8,37,59.2,35.5,54.2z"/>
+	<path class="st3" d="M45.8,53.5c2.5,0.1,4.9,0.3,7.4,0.8c-1.5,4.9-4,9.5-7.4,13.3C45.8,62.9,45.8,58.2,45.8,53.5z"/>
+	<path class="st3" d="M24,57.7c2.8-1.2,5.7-2.2,8.7-2.9c1.1,3.9,2.9,7.5,5.1,10.9C33.2,63,28.6,60.4,24,57.7z"/>
+	<path class="st3" d="M56,54.8c3,0.7,5.9,1.7,8.7,2.9c-4.6,2.7-9.3,5.3-13.9,8C53.1,62.4,54.8,58.7,56,54.8z"/>
+	<path class="st3" d="M75.5,61.1c1.3-0.6,2.8,0.6,2.6,2c-0.1,1.4-1.9,2.2-3,1.3C73.8,63.7,74.1,61.6,75.5,61.1z"/>
+</g>
+</svg>
```
Comment (fritzmg) [inline meta/numero2/contao-opengraph3/logo.svg]: Please use a minifier as mentioned in the README.

**Comment by bennyborn** [comment]
> The whitelist contains more changes than necessary. Please create a separate PR, if you want to change or re-order existing whitelist entries.

Sorry, I thought the whitelist was meant to be sorted alphabetically.

**Comment by fritzmg** [comment]
> Sorry, I thought the whitelist was meant to be sorted alphabetically.

It is, but it should be done separately :)

**Comment by bennyborn** [comment]
In this case... So I should create a separate PR just to add one word to the whitelist too?

**Comment by fritzmg** [comment]
> In this case... So I should create a separate PR just to add one word to the whitelist too?

No, everything related to this PR should stay in this PR. Re-ordering of existing whitelist entries is not related to this PR. But you can create a second PR to re-order the whitelist, if you want.

**Pull request merged**